src/net/sourceforge/kolmafia/textui/RuntimeLibrary.java

```diff
@@ -5196,7 +5196,13 @@ public static Value faxbot(ScriptRuntime controller, final Value monsterName) {
 
     FaxBotDatabase.configure();
 
-    for (FaxBot bot : FaxBotDatabase.faxbots) {
+    // Get the faxbot the user last looked at
+    int preferedBot = Math.max(Preferences.getInteger("faxbots"), 0);
+
+    // Iterate from 0
+    for (int i = 0; i < FaxBotDatabase.faxbots.size(); i++) {
```

Comment:
Alternatively you could start at the preferred index, increment and modulo and then use getting the start index as the exit condition.  I think that would remove the modulo from getFaxbot and make it a little clearer what is going on and why.